kernel/Kconfig

```diff
@@ -319,6 +319,13 @@ config BOOT_DELAY
 	  achieved by waiting for DCD on the serial port--however, not
 	  all serial ports have DCD.
 
+config BOOT_DELAY_RANDOM
+	bool "Randomize boot delay"
+	help
+	  This option randomizes the bootup delay between 0 and the
+	  value of BOOT_DELAY. This can be used e.g., to jitter boot-up
+	  of wireless network devices.
```

Comment:
It is worth to add a note about it in `BOOT_DELAY` option. Because there is saying that the entire value set will be used to delay the boot.

Reply:
If BOOT_DELAY_RANDOM is not set, then BOOT_DELAY is used.

Would you rather add a random delay on top of BOOT_DELAY?